assets/queries/cloudFormation/accessKey_not_rotated_within_90_days/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "Access_Key_Is_Not_Rotated_Within_90_Days",
+  "queryName": "Access Key Is Not Rotated Within 90 Days",
+  "severity": "MEDIUM",
+  "category": "Identity and Access Management",
+  "descriptionText": "Check if there isn't a rule that enforces access keys to be rotated within 90 days.",
+  "descriptionUrl": "https://docs.amazonaws.cn/en_us/config/latest/developerguide/access-keys-rotated.html"
+}
```

Comment:
[As a good POSIX practice](https://unix.stackexchange.com/a/18789) add end-of-file newline